scheduler/plugin/guest.go

```diff
@@ -238,6 +242,21 @@ func (g *guest) unreserve(ctx context.Context) {
 	}
 }
 
+// permit calls guestExportPermit.
+func (g *guest) permit(ctx context.Context) (*framework.Status, time.Duration) {
+	defer g.out.Reset()
+	callStack := g.callStack
+
+	if err := g.permitFn.CallWithStack(ctx, callStack); err != nil {
+		return framework.AsStatus(decorateError(g.out, guestExportPermit, err)), 0
+	}
+
+	statusCode := int32(callStack[0] >> 32)
+	timeoutMilliSeconds := int32(callStack[0])
```

Comment:
How about using uint32 here, just to prevent a too big positive number from changing to a negative number?